templates/Lambda4AppSyncHTTP/index.mjs

```diff
@@ -1,7 +1,8 @@
 import axios from "axios";
 import * as rax from 'retry-axios';
 import { aws4Interceptor } from "aws4-axios";
-import {resolveGraphDBQueryFromAppSyncEvent, refactorGremlinqueryOutput} from './output.resolver.graphql.js';
+import pkg from './output.resolver.graphql.js';
+const {resolveGraphDBQueryFromAppSyncEvent, refactorGremlinqueryOutput} = pkg;
```

Comment:
This change shouldn't be needed if https://github.com/Bit-Quill/amazon-neptune-for-graphql/pull/9/files#r1878889657 is fixed